### PR TITLE
Coming soon: Add coming soon settings to list of site options that should be sync'd

### PR DESCRIPTION
**Pull request by p-jackson**
#### Changes proposed in this Pull Request

* Attempt to sync coming soon site options between simple and atomic sites using jetpack

#### Testing instructions

*

Fixes #

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php [hunk 1]
```diff
@@ -129,3 +129,21 @@ function coming_soon_page() {
...
+ *
+ * @param array $list Existing options allowlist.
+ * @return array Updated options allowlist.
+ */
+function add_sync_options_allowlist( $list ) {
+	return array_merge(
+		$list,
+		array(
+			'wpcom_public_coming_soon',
+			'wpcom_public_coming_soon_page_id',
```
Comment (ramonjd) [inline apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php:145]: We don't need `wpcom_public_coming_soon_page_id` any more

**Comment by ramonjd** [comment]
@adamziel @jblz @michaeldcain Sorry to ruin your days here and/or risk terminal déjà vu, but we'd really love a little guidance/clarity on the minimum we need to do to:

- ensure an option (`wpcom_public_coming_soon`) is transferred to Atomic sites
- ensure this option is synched properly when toggled

As you know we're decoupling coming soon from the privacy model.

I've seen D41130-code and it seems to be the right direction. 

Are the changes in this PR relevant or even required?

Thanks for any advice you can give!

**Comment by adamziel** [comment]
@ramonjd There's plenty of internal details involved so I replied in pbxlJb-sY-p2 instead.

**Comment by p-jackson** [comment]
This has been implemented in other PRs

**Pull request closed without merging**